taichi/backends/opengl/codegen_opengl.cpp

```diff
@@ -14,14 +14,16 @@
 TLANG_NAMESPACE_BEGIN
 namespace opengl {
 
-KernelParallelAttrib::KernelParallelAttrib(OffloadedStmt *stmt)
-    : KernelParallelAttrib(-1) {
+ParallelSize_DynamicRange::ParallelSize_DynamicRange(OffloadedStmt *stmt) {
```

Comment:
nit: Could you put these ctors to `opengl_api.cpp` so that all these subclassses are placed in one place?